index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="/css/style.css" />
+        <title>ToDo App</title>
+    </head>
+    <body>
+        <header class="header">
+            <div class="header__container">
+                <nav class="header__nav">
+                    <div class="header__logo">
+                        <div class="header__logo-img">
+                            <img src="/images/leaf-1.png" alt="leaf" />
+                        </div>
+                        <div class="header__logo-text">ToDos</div>
+                    </div>
+                    <div class="header__search-input">
+                        <input type="text" placeholder="search todo" />
+                    </div>
+                </nav>
+            </div>
+        </header>
+        <section class="task-board">
+            <div class="task-board__container">
+                <h1 class="task-board__title">Add Tasks</h1>
+                <div class="task-board__button-area">
+                    <button
+                        class="task-board__button task-board__button--create"
+                    >
+                        +Create Task
+                    </button>
+                    <div class="task-board__button-wrapper">
+                        <button class="task-board__button">All</button>
```

Comment:
https://github.com/mza-wtag/todo-app-vanilla-js/pull/1#discussion_r1414950769

Here, I told to not use same classname repeatedly where every button is under the same parent div. I suggested to change the button styles using the parent `div` element.